FindMe/Model/User.m

```diff
@@ -10,15 +10,30 @@
 
 @implementation User
 
-    @dynamic firstName;
-    @dynamic lastName;
-    @dynamic email;
-    @dynamic password;
-    @dynamic profileImage;
+@dynamic firstName;
+@dynamic lastName;
+@dynamic username;
+@dynamic email;
+@dynamic password;
+@dynamic profileImage;
 
 + (nonnull NSString *)parseClassName
 {
     return @"User";
 }
 
++ (void)createUser: (NSDictionary *)userDetails  withCompletion: (PFBooleanResultBlock  _Nullable)completion{
```

Comment:
This is a database operation. Should be something like `saveUser:` and be on the DatabaseManager.